core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -69,6 +69,8 @@ private[spark] object CoarseGrainedClusterMessages {
       resources: Map[String, ResourceInformation])
     extends CoarseGrainedClusterMessage
 
+  case class ExecutorConstructed(executorId: String) extends CoarseGrainedClusterMessage
```

Comment:
Maybe `ConstructedExecutor` or `LaunchedExecutor`? To keep the same naming policy with the rest of the messages.

Reply:
thanks, changed